src/ThirdParty/Gitlab/Adapter/GitLabIssueTracker.php

```diff
@@ -190,7 +196,18 @@ public function getComments($id)
             $this->client->api('issues')->show($this->getCurrentProject()->id, $id)
         );
 
-        return $issue->showComments();
+        $comments = [];
+        $issue->showComments();
```

Comment:
Hum, after reading the code of Gitlab API model more precisely it seems that you are right. Get data will simplify the code :) So the php-gitlab-api PR is not useless but not really necessary for our case.